src/components/ItemList.svelte

```diff
@@ -96,10 +128,11 @@
         </tr>
       </thead>
       <tbody>
-        {#each filteredItems as item}
+        {#each pagedItems as item}
           <tr>
             <td>
               <a href={getItemURL(appName, itemType, item.name)}>{item.name}</a>
+              {#if isExpired(item.expires)}<i>(expired)</i>{/if}
```

Comment:
Just curious, what was the rationale for this style as opposed to the pills? Do you think it looks better?

Reply:
I wanted to keep the table items looking minimal, but now that I think about it more I don't think using Pills would clutter it too much. Do you think using Pills might be better? 

<img width="1423" alt="Screen Shot 2021-01-26 at 7 59 02 AM" src="https://user-images.githubusercontent.com/28797553/105871009-b5434e00-5fad-11eb-82c2-6343726dc620.png">
<img width="1422" alt="Screen Shot 2021-01-26 at 7 59 45 AM" src="https://user-images.githubusercontent.com/28797553/105871012-b6747b00-5fad-11eb-8deb-ea857c783bff.png">

Reply:
I'd probably vote in favour of pills, just to be consistent with the rest of the application.

Reply:
👍